src/pool/srv_pool.c

```diff
@@ -925,14 +928,52 @@ handle_event(struct pool_svc *svc, struct pool_svc_event *event)
 		goto out_prop;
```

Comment:
Is this where we should check if "auto reint" is enabled?

    if (a dead event && auto exclude is off) or (an alive event && auto reint is off)
        ignore this event

Note that this PR currently ignores an alive event is "auto exclude" is off, which doesn't seem right?